react/src/components/masthead/components/SprkMastheadSelector/SprkMastheadSelector.test.js

```diff
@@ -28,7 +28,7 @@ describe('SprkMastheadSelector:', () => {
     };
```

Comment:
verify if this is added to spark exports @RVMendoza

Reply:
Confirmed.